tests/framework/ROM/pickleTests/gold/StochasticPolyPickleTest/Udump.csv

```diff
@@ -1,37 +1,37 @@
-x2,x1,ans
-1.0,1.0,2.0
-1.8,1.0,2.8
-2.6,1.0,3.6
-3.4,1.0,4.4
-4.2,1.0,5.2
-5.0,1.0,6.0
-1.0,1.8,2.8
-1.8,1.8,3.6
-2.6,1.8,4.4
-3.4,1.8,5.2
-4.2,1.8,6.0
-5.0,1.8,6.8
-1.0,2.6,3.6
-1.8,2.6,4.4
-2.6,2.6,5.2
-3.4,2.6,6.0
-4.2,2.6,6.8
-5.0,2.6,7.6
-1.0,3.4,4.4
-1.8,3.4,5.2
-2.6,3.4,6.0
-3.4,3.4,6.8
-4.2,3.4,7.6
-5.0,3.4,8.4
-1.0,4.2,5.2
-1.8,4.2,6.0
-2.6,4.2,6.8
-3.4,4.2,7.6
-4.2,4.2,8.4
-5.0,4.2,9.2
-1.0,5.0,6.0
-1.8,5.0,6.8
-2.6,5.0,7.6
-3.4,5.0,8.4
-4.2,5.0,9.2
-5.0,5.0,10.0
+x1,x2,ans
+1.0,1.0,1.999999999999986
+1.0,1.8,2.7999999999999923
```

Comment:
this is quite interesting... O_o

Reply:
You mean the increased precision printing?  Pandas prints more digits by default than Numpy, so you see some of the numerical roundoff.  I'm sure we can change this if we want to.